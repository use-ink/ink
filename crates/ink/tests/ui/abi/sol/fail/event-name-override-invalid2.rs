#[ink::event(name = "My-Event")]
pub struct Event {
    #[ink(topic)]
    pub topic: [u8; 32],
    pub field_1: u32,
}

fn main() {}
