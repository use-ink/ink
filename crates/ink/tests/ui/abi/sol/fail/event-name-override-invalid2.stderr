error: `name` attribute argument value can only contain alphanumeric characters, underscores and dollar signs
 --> tests/ui/abi/sol/fail/event-name-override-invalid2.rs:1:21
  |
1 | #[ink::event(name = "My-Event")]
  |                     ^^^^^^^^^^
